docs/config_search.md

```diff
@@ -303,6 +304,35 @@ After Model Analyzer has found the best config(s), it will then sweep the top-N
 
 ---
 
+## LLM Search
+
+_This mode has the following limitations:_
+
+- Summary/Detailed reports do not include the new metrics
+
+LLMs can be optimized using either Quick or Brute search mode by setting `--model-type LLM`. You can specify CLI options to the GenAI-Perf tool using `genai_perf_flags`. See the [GenAI-Perf CLI](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/genai-perf/README.md#cli) documentation for a list of the flags that can be specified.
```

Comment:
This reads a little funny. I read "using either Quick or Brute search" and then I read "`--model-type LLM`" and get confused thinking, "isn't the option supposed to say quick or brute search since that's what was just said in the words before?"

Reply:
Does this statement indicate "LLMs can be optimized using either Quick or Brute search mode by setting"  means optimizing the models itself?
Should we make it more precise by stating "LLMs deployment can be optimized using either Quick or Brute search mode by setting" or something similar?

Reply:
I've rewritten the paragraph for clarity.

Reply:
@ganeshku1 I'm using the same verbiage as the other modes in config search. I would hope that users who are reading this paragraph are already familiar enough with the tool to understand that MA does run-time configuration optimizations.